lib/singleton.rb

```diff
@@ -93,21 +93,25 @@
 #
 module Singleton
   VERSION = "0.1.1"
+  VERSION.freeze
```

Comment:
Is this intentional? I think `VERSION = "0.1.1".freeze` is the correct change.

Reply:
Why aren't we just using `frozen_string_literal: true`?

Reply:
Thank you both! I meant to clean that up. I've changed it, and it should be fixed now!

Reply:
👍